authors/apps/authentication/tests/test_login.py

```diff
@@ -0,0 +1,46 @@
+
+from rest_framework import status
+from authors.base_file import BaseTestCase
+
+
+class TestAuthentication(BaseTestCase):
+    """Test class for the authentication"""
+
+
+    def test_user_login(self):
+        """test login with accurate data"""
+        self.client.post(
+            self.register_url,
+            self.register_data,
+            format="json"
+        )
+        response = self.client.post(
+            self.login_url,
+            self.login_data,
+            format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_user_login_with_no_data(self):
+        """test login with invalid data"""
+        data = {
```

Comment:
Your PR says you're testing comments. Why are you testing auth here? Make sure your PR only includes your changes

Reply:
I've done that.